### ISSUE TITLE
Is there an .nccrc.json file or something?

### ISSUE THREAD

**Issue by jbcpollak** [zeit/ncc#369]
I see in the documentation that when you call ncc programmatically you can pass in options like explicit externals, and there is some control from the command line too. Can we configure these options via a config file of some kind as well?

**Comment by styfle** [comment]
Yes run `ncc help` to see the CLI options.

https://github.com/zeit/ncc/blob/master/src/cli.js#L21

**Comment by jbcpollak** [comment]
But can those options be put in a json file instead of on the cli?

**Comment by styfle** [comment]
No, `ncc` doesn't support `json` config files.

You can create your own script to support this:

```js
const config = require('ncc-config.json');
const ncc = require('ncc');
const { code, map, assets } = await ncc('src/index.js', config);
```

**Comment by alex-page** [comment]
@styfle Would it be useful to add these values to the README.md file? I wanted to see them at a quick glance and couldn't see them.

**Comment by styfle** [comment]
It is in the readme: https://github.com/zeit/ncc#programmatically-from-nodejs

**Comment by alex-page** [comment]
Oh I meant:
```
Options:
  -o, --out [file]         Output directory for build (defaults to dist)
  -m, --minify             Minify output
  -C, --no-cache           Skip build cache population
  -s, --source-map         Generate source map
  --no-source-map-register Skip source-map-register source map support
  -e, --external [mod]     Skip bundling 'mod'. Can be used many times
  -q, --quiet              Disable build summaries / non-error outputs
  -w, --watch              Start a watched build
  --v8-cache               Emit a build using the v8 compile cache
```

**Comment by styfle** [comment]
Fixed in #478
